docs/iov-name-service/clients/00-iov-core.md

```diff
@@ -0,0 +1,179 @@
+const chainId = connection.chainId();
+```
+
+> *chainId should print ‘iov-lovenet’*
+
+a wallet and an iov address connected to our profile. we will generate a random mnemonic
+
+```
+const randomEntropy = Random.getBytes(32);
+const wallet = profile.addWallet(Ed25519HdWallet.fromEntropy(await randomEntropy));
+const myIdentity = await profile.createIdentity(wallet.id, chainId, HdPaths.iov(1));
+const myAddress = signer.identityToAddress(myIdentity);
+console.log("secret random mnemonic: ", profile.printableSecret(wallet.id));
+```
+
+If you want to know my secret (doo da do), it was:
+
+```
+company buddy electric crouch boss noble lift machine other unfair toilet pill barely mass sunny beach lamp release ancient call zoo mask grit puppy
+```
+
+and myAddress:
+```
+tiov1azf4469g720ea3pzgtctz7tm9ema7kgft7pyqf
+```
+
+## Get some IOV tokens on my account
+
+We are connected to iov-lovenet (testnet), so we can use iov faucet to get some tokens
+
+    const faucet = new IovFaucet("https://bns-faucet.lovenet.iov.one/");
+
+    await faucet.credit(myAddress, "IOV" as TokenTicker);
+
+Check that the account has some tokens now
+
+    let myAccount = await connection.getAccount({ address: myAddress });
+
+    myAccount
+    >> { address: 'tiov1azf4469g720ea3pzgtctz7tm9ema7kgft7pyqf',
+      balance:
+       [ { quantity: '10000000000',
+           fractionalDigits: 9,
+           tokenTicker: 'IOV' } ],
+      pubkey: undefined }
+
+## Register a Name in IOV Name Service
+
+Now is time to use one of the iov value propositions: a human readable address that works as a Non Fungible Token and allows storage of multiple addresses from multiple blockchains
+
+First we need to create the transaction body. You can add as many chainId/address pairs a you like, following the list template below:
+> *use your username here, mine is already taken ;)*
+
+    .editor
+
+    const registrationTx = await connection.withDefaultFee<RegisterUsernameTx & WithCreator>({
+      kind: "bns/register_username",
+      creator: myIdentity,
+      targets: [
+        { chainId: chainId, address: myAddress },
+        { chainId: "cosmos-hub2" as ChainId, address: "cosmos17w5kw28te7r5vn4qu08hu6a4crcvwrrgzmsrrn" as Address}
+      ],
+      username: "antoine*iov",
+    });
+    ^D
+
+Now we sign, post and confirm the transaction
+
+    await signer.signAndPost(registrationTx);
+
+    const bnsConnection = connection as BnsConnection;
+
+    const myAccountBNS  = await bnsConnection.getUsernames({ owner: myAddress });
+    >> [ { id: 'lucas',
+        owner: 'tiov188ayx37py2r86wz5a4a2vrn4ejrwhnnte4n7kc',
+        targets: [ [Object], [Object] ] } ]
+
+    myAccountBNS[0].targets
+
+    >> [ { chainId: 'iov-lovenet', address: 'tiov1azf4469g720ea3pzgtctz7tm9ema7kgft7pyqf' }, { chainId: 'cosmos-hub2', address: 'cosmos17w5kw28te7r5vn4qu08hu6a4crcvwrrgzmsrrn' } ]
+
+That is it! Welcome to the world of Personalized Names :)
+
+## Extra Mile: Send IOV Tokens to a Personalized Name
```

Comment:
Capitalization on the sections is inconsistent.  eg, sometimes it's "IOV tokens" and others it's "IOV tokens".